source/documentation/10-switching-to-production.md

```diff
@@ -91,7 +89,7 @@ Go to _Configuration > Technical information_
 1. On the _Technical information_ page, click the _Data and origin verification_ tab.
 1. Scroll to the _Checks for e-Commerce & Alias Gateway_ section.
 1. Leave the _URL of the merchant page containing the payment form that will call the page: orderstandard.asp_ field blank.
-Enter a strong SHA-IN passphrase; this will be used when setting up the GOV.UK account credentials.
+1. Enter a strong plaintext string SHA-IN passphrase; this will be used when setting up the GOV.UK account credentials.
```

Comment:
Not too sure about this: I’ve only ever seen “plaintext” (with no spaces) used in contrast to “ciphertext” (i.e. unencrypted versus encrypted text); not sure it’s appropriate in this context.

I’m also a bit concerned that the extra words are taking “strong” too far away from “passphrase” (it almost seems like it’s a modifier for “string”, which we may be able to drop).

And I’ve just noticed that it says “GOV.UK account credentials” rather than “GOV.UK Pay account credentials”, which we may as well fix while we’re here.

Could we go with something like this?

“Enter a strong _SHA-IN passphrase_ (plain text, not a hash); this will be used when setting up the GOV.UK Pay account credentials.”

Reply:
Agree, I've made the changes